tests/frontier/scenarios/programs/invalid_opcodes.py

```diff
@@ -0,0 +1,48 @@
+"""Define programs that can not be run in static context."""
+
+import pytest
+
+from ethereum_test_tools import Bytecode
+from ethereum_test_tools.vm.opcode import Opcodes as Op
+
+from ..common import ProgramResult
+
+invalid_opcode_ranges = [
+    range(0x0C, 0x10),
+    range(0x1E, 0x20),
+    range(0x21, 0x30),
+    range(0x4B, 0x50),
+    range(0xA5, 0xF0),
+    range(0xF6, 0xFA),
+    range(0xFB, 0xFD),
+    range(0xFE, 0xFF),
+]
```

Comment:
I feel like this should not be a static list, and rather should be derived from the fork properties.

Reply:
yes this can be built depending on fork

Reply:
ah, but the programs does not know about fork. I would have to refactor all programs to take fork as an argument.

Reply:
hm I can make this a predeployed contract in scenarios if you concern so much. but making programs a fixtures that accept fork is rather difficult design decision. programs are intended as byte sequences that are to be deployed in different scenarios and tested

Reply:
fixed invalid program